tests/headlessBrowserTestRunner.js

```
@@ -0,0 +1,150 @@
+export const browserEngine = process.argv[2]?.toLowerCase() ?? "chrome";
+export const plainTextHeaders = {
+	"Content-Type": "text/plain"
+};
+
+/**
+ * @param {function(puppeteer.Page): Promise<boolean>} testBody
+ * @param {function(http.IncomingMessage): Promise<void>} [httpReqFunc]
+ */
+export async function test(testBody, httpReqFunc) {
+	let indexPage = await fs.promises.readFile(path.join(import.meta.dirname, "../dist/index.html"), "utf-8");
+	let headImportMap = indexPage.match(/<script type="importmap">[^]+?<\/script>/)[0];
```

Biome (1.9.4)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)